cmd/server-manager/typescript/src/RaceControl.ts

```diff
@@ -116,6 +116,49 @@ export class RaceControl {
             case EventNewSession:
                 this.showTrackWeatherImage();
                 break;
+            case EventChat:
+                let $chatContainer = $("#chat-container");
+
+                let chatMessage = $("#chat-message-template").first().clone();
+                let chatMessageSender = $(document.createElement("SPAN"));
+
+                let dt = new Date(message.Message.Time);
+
+                let minutes = dt.getMinutes()
+                let minutesString = ""
+
+                let hours = dt.getHours()
+                let hoursString = ""
+
+                if (minutes < 10) {minutesString = "0"+minutes;} else {
+                    minutesString = minutes.toLocaleString()
+                }
+
+                if (hours < 10) {hoursString = "0"+hours;} else {
+                    hoursString = hours.toLocaleString()
+                }
+
+                chatMessageSender.attr(
+                    "style", "color: " + randomColorForDriver(message.Message.DriverGUID)
+                ).text(
+                    hoursString + ":" + minutesString + " " + message.Message.DriverName + ": "
+                )
+
+                chatMessage.text(message.Message.Message)
+                chatMessage.addClass("chat-message")
+                chatMessageSender.addClass("chat-message-sender")
+
+                $chatContainer.append(chatMessageSender)
+                $chatContainer.append(chatMessage)
+
+                if ($chatContainer.find(".chat-message").length > 20) {
```

Comment:
do we wanna keep the 20 limit for active sessions too? I don't see any harm in giving them more scrollback?